.github/workflows/go.yml

```diff
@@ -42,3 +42,18 @@ jobs:
         GO111MODULE: on
       run: make test-stdlib test-bash
 
+    - name: GitHub Actions Env Test Setup
```

Comment:
I was going to create a separate workflow file for this test but decided to just append the tests here instead. I can split it to different files if that makes more sense.

Should I "port" this test to do something when running locally on a dev machine? It feels kind of odd to only test some functionality in CI vs locally first.